### PR TITLE
Feature/gcb 52 integrate with google analytics

### PR DESCRIPTION
**Pull request by michallaskowski**
Works with Google Analytics, integration tested for page views report (test not included because of credentials, but it is very simple)

### CODE REVIEW

File: GrandCentralBoard/Extensions/NSDate+DateFormatter.swift [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+private let yearMonthDayFormatter: NSDateFormatter = {
+    let formatter = NSDateFormatter()
+    formatter.dateFormat = "yyyy-MM-dd"
+    formatter.locale = NSLocale(localeIdentifier: "en_US_POSIX")
+    return formatter
+}()
+
+extension NSDate {
+    func yearMonthDayString() -> String {
```
Comment (Augustyniak) [inline GrandCentralBoard/Extensions/NSDate+DateFormatter.swift]: I think that name should indicate that this should be used only for parsing dates from the network responses (as you override locale) - formatter used for formatting user facing strings shouldn't override locale.

**Comment by Augustyniak** [comment]
I have no more comments for now. Please rebase branch.

**Comment by Augustyniak** [comment]
...and fix mentioned issue :)

**Comment by michallaskowski** [comment]
The issue is fixed, I am merging because Rafal is not available.

**Pull request merged**